### PR TITLE
Tech/dfs 148 unit testing

### PR DESCRIPTION
**Pull request by AnHere**

### CODE REVIEW

File: lib/types/api/dio_response.dart [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class DioResponse {
```
Comment (TilDoesCode) [inline lib/types/api/dio_response.dart]: Can this be deleted?

File: lib/widgets/cards/tradablestockcard_widget.dart [hunk 2]
```diff
@@ -178,7 +179,7 @@ class TradableStockCard extends HookWidget {
                 child: Icon(
                   Icons.error_outline,
                   size: 55,
-                  color: NeoTheme.of(context)!.negativeColor,
+                  color: Colors.white,
```
Comment (TilDoesCode) [inline lib/widgets/cards/tradablestockcard_widget.dart]: We should not be hardcoding colors

File: lib/types/api/dio_response.dart (hunk 1 above)
Comment (AnHere) [reply lib/types/api/dio_response.dart]: Oh, yeah you are right!

File: lib/widgets/cards/tradablestockcard_widget.dart (hunk 2 above)
Comment (AnHere) [reply lib/widgets/cards/tradablestockcard_widget.dart]: You are right, I totally forgot this I still need a solution for this. I will think about it

Comment (AnHere) [reply lib/widgets/cards/tradablestockcard_widget.dart]: Done

**Pull request merged**